test/Microsoft.AspNet.Razor.Runtime.Test/TagHelpers/TagHelperDescriptorFactoryTest.cs

```diff
@@ -2134,5 +2211,17 @@ private class DictionaryOfIntSubclass : Dictionary<int, string>
         private class GenericDictionarySubclass<TValue> : Dictionary<string, TValue>
         {
         }
+
+        [OutputElementHint("strong")]
+        private class OutputElementHintTagHelper : TagHelper
+        {
+        }
+
+        [TargetElement("a")]
+        [TargetElement("p")]
+        [OutputElementHint("div")]
+        private class MultiOutputElementHintTagHelper : TagHelper
```

Comment:
Confusing name requires reader to review bottom of the file before they can understand the data set near the top. This `TagHelper` doesn't have multiple associated `[OutputElementHint]` attributes. It's more like `TagHelperWithMultipleTargetElementsAndAnOutputElementHint` or perhaps `MulitpleDescriptorTagHelperWithOutputElementHint`.